src/ChainSafe.GamingSDK.EVM.Tests/ChainsafeRPCTests.cs

```diff
@@ -133,43 +159,61 @@ public void GetGasPriceTest()
         [Test]
         public void GetTransactionReceiptTest()
         {
-            var latestBlockWithTx = infuraProvider.GetBlockWithTransactions().Result;
+            var latestBlockWithTx = web3.RpcProvider.GetBlockWithTransactions().Result;
             var firstTransactionInBlock = latestBlockWithTx.Transactions[0];
             var receipt =
-                infuraProvider.GetTransactionReceipt(
+                web3.RpcProvider.GetTransactionReceipt(
                     firstTransactionInBlock.Hash).Result;
             Assert.AreEqual(firstTransactionInBlock.Hash, receipt.TransactionHash);
             Assert.AreEqual(firstTransactionInBlock.BlockHash, receipt.BlockHash);
         }
 
         /*
+         * To run this test you need to deploy a contract using your Ganache instance and the contract needs to have a safeMint function that increments the tokenId counter
```

Comment:
We should tag @juans-chainsafe there probably.

Reply:
yeah, this one was the most difficult to migrate, not sure if we can automate it all because it requires a lot of manual steps

Reply:
This is actually Juan's code. I do think it should be possible to use the SDK itself to deploy a smart contract in the setup code; but I haven't verified this.

Reply:
In any case you can use nethereum to deploy contract. It should be less than 5 loc

https://docs.nethereum.com/en/latest/contracts/deploying/

Reply:
The final solution is to run `anvil` for the test execution automation.